packages/slate-dev-benchmark/src/components/Repository.js

```diff
@@ -0,0 +1,68 @@
+/* global Promise */
+const { RepositoryType } = require('./types')
+const { logger } = require('../logger')
+const { compose } = require('../compose')
+
+/* Repository Class for holding Suite
+*/
+
+class Repository {
```

Comment:
Repository is used to gather all suites results.  Because each package has one suite, then we may have different suites.  We can gather all suites results here, or gather all suites results in `benchmark/index.js`

Because repostory gathering results from suites and suite gathering result from benches are similar logic (if a bench or a suite has an error, then we log out the error and run the next bench/suite) , so I think it is easily to gather result here to get the similar logic in `./compose` together.